.github/workflows/build-example-workflows.yml

```diff
@@ -0,0 +1,22 @@
+name: Build GenSX Example Workflows
+
+on:
+  pull_request:
+    paths:
+      - "examples/client-side-tools/gensx/**"
+
+jobs:
+  build-client-side-tools:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
Should we do something like this so we can reduce redunancy? Only tricky bit is that web apps vs basic examples have a slightly different file path `src/workflow.ts` vs `gensx/workflow.ts`

```
    strategy:
      matrix:
        path: [deep-research, client-side-tools]
```